app/Util/Helpers.hs

```diff
@@ -48,3 +48,7 @@ tryReplace :: (a -> Bool) -> a -> [a] -> (Bool, [a])
 tryReplace _ _ [] = (False, [])
 tryReplace f e (h : l) = if f h then (True, e : l) else second (h :) (tryReplace f e l)
 
+cnv :: [a] -> [(a, a)]
```

Comment:
This name is a bit meaningless, maybe call this `groupTuples` or something like that?

Reply:
I copied the function from a stackoverflow question lol. I just assumed it was some special mathy name. `groupTuples` makes sense